internal/cli/push.go

```diff
@@ -35,7 +73,12 @@ var CmdPush = &cobra.Command{
 			return errors.Wrap(err, "path not accessible")
 		}
 
-		err = client.Push(args[0], path)
+		services, err := cmd.Flags().GetStringSlice("bind")
```

Comment:
And I'm doing my own thing here: https://github.com/epinio/epinio/compare/368-application-scale#diff-9d5877b4c4b9701570001c32b04a95d163025ade9671e9c5add3a32d98934e9dR38-R45 :D  

Since your PR is ready, I will rebase when we merge this and come up with some refactoring to work for all flags. Maybe we can pair on that after this one is merged.

Reply:
Heh. Sure.